spectractor/fit/mcmc.py

```diff
@@ -956,12 +956,13 @@ def run_emcee(mcmc_fit_workspace, ln=lnprob):
             sampler.run_mcmc(p0, nsteps=max(0, nsamples - backend.iteration), progress=True)
         pool.close()
     except ValueError:
+        nthread = int(os.environ.get("OMP_NUM_THREADS", multiprocessing.cpu_count()))
```

Comment:
How is OMP_NUM_THREADS getting set? Is Jenkins setting it?

Reply:
It is part of our standard jenkins-ing, and I'm just using that as a short-hand.